variables.tf

```diff
@@ -93,8 +93,8 @@ variable "memory_size" {
 
 variable "timeout" {
   type        = number
-  default     = 3
-  description = "The amount of time your Lambda Function has to run in seconds. Defaults to 3."
+  default     = 10
+  description = "The amount of time in seconds your Lambda Function will run. Defaults to 3."
```

Comment:
Pls update description, Default to 10

Reply:
- Actually when we create Lambda Function from console then the default `timeout` comes is 3 second. Above description is showing that `timeout`. 
- Updating it to `10` just to be sure that Lambda-Function is failing or stopping as sometimes Lambda fails within `3` seconds after execution.

This will not increase cost of Lambda Function 
Ref: https://stackoverflow.com/questions/70197483/long-timeout-for-aws-lambda-function